entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -e
+
+# Use /credentials as AWS credentials file if it exists
+test -f /credentials && export AWS_SHARED_CREDENTIALS_FILE="/credentials"
+
+if [[ -z "$AWS_SHARED_CREDENTIALS_FILE" ]]; then
+    echo "Either AWS_SHARED_CREDENTIALS_FILE or /credentials file must be set"
+    exit 1
+fi
+
+DRY_RUN=${DRY_RUN:-"True"}
+ACTION=${ACTION:-"Apply"}
+
+if [[ $DRY_RUN != "True" ]] && [[ $DRY_RUN != "False" ]]; then
+    echo "Invalid DRY_RUN option: $DRY_RUN. Must be 'True' or 'False'"
+    exit 1
+fi
+
+if [[ $ACTION != "Apply" ]] && [[ $ACTION != "Destroy" ]]; then
+    echo "Invalid ACTION option: $ACTION. Must be 'Apply' or 'Destroy'"
+    exit 1
+fi
+
+echo "Starting: ACTION=$ACTION with DRY_RUN=$DRY_RUN"
+
+# Terraform output options
+export TF_CLI_ARGS=${TF_CLI_ARGS:-"-no-color"}
+TERRAFORM_OUT_DIR="./module"
+TERRAFORM_CMD="terraform -chdir=$TERRAFORM_OUT_DIR"
+TERRAFORM_VARS="-var-file=tfvars.json"
+OUTPUTS_FILE="${TERRAFORM_OUT_DIR}/outputs.json"
+
+# Generate module configuration
+python er_aws_kms
```

Comment:
Could you refactor that? Maybe something like

```
generate-tf-files
```

and having a script in `pyproject.toml` in the sub project:

```
[project.scripts]
generate-tf-files = 'er_aws_kms.__main__:generate-files'
```

Reply:
Added. I also added helper methods in external-resources-io package to generate these files.